api/api/urls.py

```diff
@@ -14,14 +14,13 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path, include
-from django.conf import settings
-from django.conf.urls.static import static
+from django.urls import path
+from django.conf.urls import include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    # path('', include('cotrip_app.urls')),
-    path('', include('accounts.urls'))
```

Comment:
Can you delete the commented out code? Should you be deleting the accounts urls here?